packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -65,15 +65,22 @@ function percentToValue(percent, min, max) {
   return (max - min) * percent + min;
 }
 
-function ensurePrecision(value, step) {
-  const stepDecimalPart = step.toString().split('.')[1];
-  const stepPrecision = stepDecimalPart ? stepDecimalPart.length : 0;
+function getDecimalPrecision(num) {
+  // This handles the case when num is very small (0.00000001), js will turn this into 1e-8.
+  // When num is bigger than 1 or less than -1 it won't get converted to this notation so it's fine.
```

Comment:
Not sure if it's clear enough. 😓

Reply:
It's good enough from my perspective, thanks!